src/main/java/com/google/impactdashboard/server/api_utilities/LogRetriever.java

```diff
@@ -60,6 +62,10 @@ public ListLogEntriesPagedResponse listAuditLogsResponse(String projectId, Strin
     ListLogEntriesRequest.Builder builder = ListLogEntriesRequest.newBuilder()
         .setOrderBy("timestamp desc").addResourceNames(resourceNameStringBuilder.toString());
 
+    if(!pageToken.equals("")) {
```

Comment:
consider this library instead. In general, it avoids any null issues, which is a big advantage.

https://guava.dev/releases/19.0/api/docs/com/google/common/base/Strings.html#isNullOrEmpty(java.lang.String)

Reply:
sure thing!